apps/events/pdf_generator.py

```diff
@@ -21,7 +21,7 @@ class EventPDF(object):
 
     def __init__(self, event):
         self.event = event
-        attendee_qs = event.attendance_event.attendees_qs
+        attendee_qs = event.attendance_event.attending_attendees_qs
```

Comment:
Super nitpick: If you refactor what field you proxy, you should probably also rename the variable that proxies it. :p